content/coronavirus_landing_page.yml

```diff
@@ -60,6 +60,18 @@ content:
       text: "Find support"
   explainer_title:
   explainer_text:
+  timeline:
+    heading: Recent and upcoming changes
+    list:
+      - heading: 1 October
```

Comment:
Are we expecting content designers to add dates like this? Or will we accept a date in any UK format and then apply formatting?

Reply:
Also, should this be "effective date" rather than "heading"?

Reply:
My plan is to just take whatever heading they set and show it without applying any formatting - Let the content designer decide on format